SpywareFilter/sections/specific.txt

```diff
@@ -23,6 +23,9 @@
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 ||googleadservices.com/pagead/conversion_async.js$redirect=noopjs,domain=ffm.to,important
 !
+||js.ad4989.co.kr/common/js/tend.js
```

Comment:
It is blocked by Base filter

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/77431/commits/0d50467d7f79bef13341fb9962f9654976087b7d